client/src/constants.ts

```diff
@@ -42,3 +42,6 @@ export const ENABLE_MDNPLUSPLUS = JSON.parse(
   process.env.REACT_APP_ENABLE_MDNPLUSPLUS ||
     JSON.stringify(process.env.NODE_ENV === "development")
 );
+
+export const DEFAULT_GEO_COUNTRY =
+  process.env.REACT_APP_DEFAULT_GEO_COUNTRY || "United States";
```

Comment:
The reason we can do that is this default value will only ever matter when you're using a Kuma `/api/v1/whoami` that isn't served via Cloudfront (e.g. local dev). 
In Dev, Stage, and Prod, it will always be a value so it never falls back on this.